.circleci/config.yml

```diff
@@ -133,12 +133,16 @@ references:
   upload_build_artifacts: &upload_build_artifacts
```

Comment:
I see this step is in `linux_package` job that runs on all branches and tags. I see that package name on tags has version, on non-tag the commit hash.

And I see the bucket is distinct from the release one. Ok.